appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1140,6 +1140,13 @@ private static int upgradeLocationSensorProperties(Map<String, JSONValue> compon
       // The speed parameter was added to the LocationChanged event
       srcCompVersion = 3;
     }
+    if (srcCompVersion < 4) {
+      // Make location sensor disable by default
+      if (!componentProperties.containsKey("Enabled")) {
+        componentProperties.put("Enabled", new ClientJsonString("True"));
+      }
```

Comment:
You could also remove the "Enabled" key if it exists and the value is "False" since this is the new default.

Reply:
Okay, I was actually thinking about this before.

Reply:
That's what I said to do :wink: